ui/app/styles/core/message.scss

```diff
@@ -106,6 +106,7 @@
 
 .message-inline {
   display: flex;
+  align-items: center;
```

Comment:
is this going to be true for all message-inline?  worth checking that this doesn't have consequences elsewhere.